### PR TITLE
New map, Oscar Outpost [MDB Ignore]

### PR DESCRIPTION
**Pull request by RipGrayson**
## About The Pull Request

Adds Oscar outpost, an alternate map to WO.
**I did not originally author this, the original author was Warfan1815 who has graciously given permission for this port**
All changes deviating from his original vision are my own.
 
trench2

Xenos charge in from the north, marine base in the south.
It has the landmarks for CP and Sensor cap in case an admin wants to RP the marines defending a beach against invading CLF forces.

## Why It's Good For The Game
An alternate to WO is nice for when admins want to mess around or play hold the line against xeno hordes. This one has more possibilities compared to regular WO, at least in terms of possible scenarios.

## Changelog
:cl:
add: Added a new alternate map to WO, Oscar outpost.
/:cl:

### CODE REVIEW

**Comment by Xsdew** [comment]
Marines, get the MG42s ready.

**Comment by Araulius** [comment]
we'll make our own wave attack map, with blackjack and hookers!

**Comment by Lumipharon** [comment]
I love the doctor evil lava lair thing the marines having going on.

>are we the baddies?

**Comment by novaepee** [comment]
> Have you noticed that our EVA helmets have actually got little pictures of skulls on on them.
> are we the baddies?

File: code/game/objects/effects/spawners/random.dm [hunk 1]
```diff
@@ -1449,6 +1487,18 @@
 		/obj/item/stack/sheet/metal = 1,
 	)
 
+/obj/effect/spawner/random/structure/m42c
```
Comment (XSlayer300) [inline code/game/objects/effects/spawners/random.dm:1490]: M42Cs don't exist in TGMC, don't they?

**Comment by SilencedMP5A5** [comment]
love it

**Comment by Losenis** [comment]
Is that all grass?

Sounds like LV if it was all colony

**Comment by RipGrayson** [comment]
> Is that all grass?
> 
> Sounds like LV if it was all colony

This map isn't intended for XvH, it's an alternate to WO and actual xeno playability wasn't a consideration.

**Comment by Lumipharon** [comment]
ebent.

**Pull request merged**